packages/airnode-deployer/src/utils/logger.ts

```diff
@@ -84,3 +90,21 @@ export function debugMode(mode: boolean) {
 export function inDebugMode() {
   return debugModeFlag;
 }
+
+export function setSecret(secret: string) {
+  secrets.push(secret);
+}
+
+export function replaceSecrets(input: string, secrets: string[]) {
+  let output = input;
+  secrets.forEach((secret) => (output = output.replace(secret, '*'.repeat(secret.length))));
+
+  return output;
+}
+
+export function setLogsDirectory(path: string) {
+  logsDirectory = path.endsWith('/') ? path.slice(0, -1) : path;
+  if (!fs.existsSync(logsDirectory)) fs.mkdirSync(logsDirectory, { recursive: true });
+
+  logFileTimestamp = format(Date.now(), 'yyyy-MM-dd_HH:mm:ss');
```

Comment:
If we're using the format at multiple places it should be extracted in a constant.